src/main/java/com/ibm/crypto/plus/provider/ock/Digest.java

```diff
@@ -343,4 +339,24 @@ protected static boolean validId(long id) {
         //OCKDebug.Msg(debPrefix, methodName,  "Id : " + id);
         return (id != 0L);
     }
+
```

Comment:
Perhaps we can add a override Annotation there.

Reply:
Done. I, also, added the annotation to `MessageDigest`.